modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,127 @@
+== Fedora CoreOS - Getting Started
+
+Fedora CoreOS has no install-time configuration. All Fedora CoreOS installs or cloud images start the same, then on first boot Ignition will read the supplied config and configure the system. Ignition configs are usually supplied via the cloud’s userdata mechanism, or, in the case of bare metal, injected at install time. This guide will show you how to launch Fedora CoreOS on AWS, QEMU, and bare metal as well as how to create Ignition configs.
+
+=== Launching Fedora CoreOS
+
+==== Booting on AWS:
+
+To boot on AWS, find the correct ami in the https://builds.coreos.fedoraproject.org/browser[release browser] and specify the Ignition config as the https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-add-user-data[user-data]. Note that while the AWS documentation mentions cloud-init and scripts, Fedora CoreOS only accepts Ignition configs; it does not support cloud-init or running scripts from userdata.
+
+. Example launching Fedora CoreOS on AWS
+[source, bash]
+aws ec2 run-instances <other options> --user-data file://config.ign 
+
+==== Booting with QEMU:
+
+Download the QEMU Fedora CoreOS image from the https://builds.coreos.fedoraproject.org/browser[release browser]
+
+Specify your Ignition config by using `-fw_cfg` to set the `opt/com.coreos/config` key in the QEMU firmware configuration device.
+
+. Example launching Fedora CoreOS with QEMU
+[source, bash]
+qemu-system-x86_64 -machine accel=kvm -m 2048 -cpu host -nographic \
+	-drive if=virtio,file=path/to/fedora-coreos-qemu.qcow2 \
+	-device virtio-rng-pci \
+	-fw_cfg name=opt/com.coreos/config,file=path/to/ignition-config.ign
+
+==== Installing on bare metal:
+
+Follow the https://github.com/coreos/coreos-installer/[coreos-installer instructions] to install Fedora CoreOS to disk. This will inject your specified Ignition config in the installed image. Note that Fedora CoreOS does not support running live PXE booted yet.
```

Comment:
> Note that Fedora CoreOS does not support running live PXE booted yet.

Is this useful?  We should just tell users to download the installer ISO and run it (with the requisite kargs).

Reply:
I thought we preferred PXE to the iso? I mean this as a "we don't support rootless PXE like container linux"

Reply:
We do prefer it, but it doesn't exist yet.  I don't think the getting-started guide is the right place for telling CL users which pieces aren't ready yet.  (Not inline, anyway.  We could have a callout box or something.)

Reply:
We can install over PXE, I've done it. We just can't run normally from pxe

Reply:
Oh, I see what you meant.  Preferring installer-PXE over installer-ISO is fine.  I just don't think we should talk about live PXE here.